attributes/remi-gpgkey.rb

```diff
@@ -0,0 +1,11 @@
+# From https://rpms.remirepo.net/ in the "Other Resources" section about
+# gpg keys for platform versions.
+default['yum-remi-chef']['gpgkey'] = if node['platform'].eql?('fedora') &&
```

Comment:
perhaps you could use the dsl method, `platform?('fedora')` here instead.

Reply:
I fixed this post merge